.github/workflows/tests.yml

```diff
@@ -18,13 +18,6 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  check:
-    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
-    with:
-      envs: |
-        - linux: check-style
-        - linux: check-types
```

Comment:
Without a replacement (perhaps adding mypy to pre-commit) we need to keep this to keep mypy.